stubs/pyinstaller/PyInstaller/compat.pyi

```diff
@@ -0,0 +1,82 @@
+# https://pyinstaller.org/en/stable/hooks.html#module-PyInstaller.compat
+from _typeshed import GenericPath, StrOrBytesPath
+from collections.abc import Iterable
+from importlib.abc import _Path
+from types import ModuleType
+from typing import AnyStr, overload
+from typing_extensions import Literal, TypeAlias
+
+_OpenFile: TypeAlias = StrOrBytesPath | int
+
+is_64bits: bool
+is_py35: bool
+is_py36: bool
+is_py37: bool
+is_py38: bool
+is_py39: bool
+is_py310: bool
+is_win: bool
+is_win_10: bool
+is_win_wine: bool
+is_cygwin: bool
+is_darwin: bool
+is_linux: bool
+is_solar: bool
+is_aix: bool
+is_freebsd: bool
+is_openbsd: bool
+is_hpux: bool
+is_unix: bool
+is_musl: bool
+is_macos_11_compat: tuple[int, ...] | bool | None
+is_macos_11_native: tuple[int, ...] | bool | None
+is_macos_11: tuple[int, ...] | bool | None
```

Comment:
They are in PyInstaller's develop branch because I did a PR over there to fix that. But as of right now with the version on PyPI this is accurate.
For instance, changing them to bool will raise the following in stubtest:
```
error: PyInstaller.compat.is_macos_11 variable differs from runtime type None
Stub: at line 33 in file stubs\pyinstaller\PyInstaller\compat.pyi
builtins.bool
Runtime:
None

error: PyInstaller.compat.is_macos_11_compat variable differs from runtime type None
Stub: at line 31 in file stubs\pyinstaller\PyInstaller\compat.pyi
builtins.bool
Runtime:
None

error: PyInstaller.compat.is_macos_11_native variable differs from runtime type None
Stub: at line 32 in file stubs\pyinstaller\PyInstaller\compat.pyi
builtins.bool
Runtime:
None
```